source/js/src/post-details.js

```diff
@@ -89,8 +89,7 @@ $(document).ready(function() {
   var display = CONFIG.page.sidebar;
   if (typeof display !== 'boolean') {
     // There's no definition sidebar in the page front-matter
-    var isSidebarCouldDisplay = CONFIG.sidebar.display === 'post'
-     || CONFIG.sidebar.display === 'always';
+    var isSidebarCouldDisplay = CONFIG.sidebar.display === 'post' || CONFIG.sidebar.display === 'always';
```

Comment:
This will brake [ESLinter under the NexT.](https://github.com/theme-next/eslint-config-theme-next). Revert this.

Reply:
Sure